docs/frequently-asked-questions.rst

```diff
@@ -273,9 +273,11 @@ of variable it concerns:
 
 * state variables are always in storage
 * function arguments are in memory by default
```

Comment:
We remove the default here, too, don't we?

Reply:
This PR doesn't - and there wasn't a warning about a location missing for those before. I was wondering, whether or not we should still remove the default here as well. I'll create an issue for it, if there isn't already one.